app/js/components/presentational/LoginForm.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
```

Comment:
All OpenMRS app files should have the license comments on top.

Reply:
Great the PR had been updated